js/src/lib/interfaces/Types.ts

```diff
@@ -718,6 +718,7 @@ export interface ModelData {
 		inference?: boolean | {
 			parameters?: Record<string, any>;
 		};
+		base_model?: string;
```

Comment:
not really needed IMO as it's quite specific

otherwise, add a comment that it's only for diffusers models?

Reply:
I added it because the tests didn't pass.